test.js

```diff
@@ -1,7 +1,12 @@
-'use strict';
-var assert = require('assert');
-var splitLines = require('./');
+import test from 'ava';
+import fn from './';
```

Comment:
`generator-nm` still defaults to `fn`: https://github.com/sindresorhus/generator-nm/blob/master/app/templates/test.js#L2